CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+- Prop `unitPriceDisplay` to `unit-price` block.
+- Prop `displayUnitListPrice` to `unit-price` block.
```

Comment:
just a minor nitpick, can you remove the empty line between "Unreleased" and "Added"?
```suggestion
## [Unreleased]
### Added
- Prop `unitPriceDisplay` to `unit-price` block.
- Prop `displayUnitListPrice` to `unit-price` block.
```